Sources/ComposableArchitecture/Store.swift

```diff
@@ -8,9 +8,7 @@ import ReactiveSwift
 /// the `scope` method to derive more focused stores that can be passed to subviews.
 public final class Store<State, Action> {
   @MutableProperty private(set) var state: State
-  var effectCancellables: [UUID: Disposable] = [:]
```

Comment:
Due to the way that ReactiveSwift Disposables work, we don't need to hold onto them here.